### ISSUE TITLE
fc2vcf.py key error

### ISSUE THREAD

**Issue by dBenedek** [ndaniel/fusioncatcher#173]
FusionCatcher gives the following error: 
```
Traceback (most recent call last):
  File "/disk/work/shared/tools/fusioncatcher-1.20/bin/fc2vcf.py", line 124, in <module>
    h_eff = header["Predicted_effect"]
KeyError: 'Predicted_effect'
```

Command used: `fusioncatcher -p {threads} -i {input.fq1},{input.fq2} -o {output}`
FusionCatcher version: 1.20, loading with module
Full log file: [fusioncatcher.log](https://github.com/ndaniel/fusioncatcher/files/5121628/fusioncatcher.log)
Probably there are no fusions found for those fastq files, that's why the intermediate CSV file doesn't have "Predicted_effect", so the fc2vcf.py fails?

**Comment by ndaniel** [comment]
I guess that this was fixed with this patch:

https://github.com/ndaniel/fusioncatcher/commit/5813f67bc075cb45fd2b2cdcfb5c836ac5e0b6ef

**Comment by ndaniel** [comment]
This will be fixed in v1.30.
